cypress/e2e/laManager/addServices/addServices-laManager-WhatIsYourServicePage.spec.js

```diff
@@ -0,0 +1,15 @@
+describe("LA Manager - Add Services - Service Added Page", () => {
+    beforeEach(() => {
+        cy.visit('/')
+        cy.integrationLogin('laman')
+
+    })
+
+    it('Ensuring that clicking on the back button on the add service page', () => {
+        cy.contains('Services').click();
+        cy.get('a[href*="/manage-services/start-add-service"]').click();
+        cy.clickBackLink();
+        cy.title().should('eq', 'Tower Hamlets Council services - Manage family support services and accounts - GOV.UK');
```

Comment:
For back button tests I personally check the URL of the resultant page matches what I expect, rather than testing content on that page (since it's out of scope for that particular test), as a suggestion :)

Reply:
Good shout! URL would be better than content or page title

Reply:
The page titles are not the same on each page; they should be different. The page title URL follows the convention:  current page name - service name - [GOV.UK] and can be used as verification. See No 8 of the acceptance criteria of FHG-5136